src/amo/components/AddonReview/index.js

```diff
@@ -100,7 +100,10 @@ export class AddonReviewBase extends React.Component<Props, State> {
     event.preventDefault();
     event.stopPropagation();
 
-    const newReviewParams = { body: reviewBody || undefined };
+    // If reviewBody text is null, we'll assign it as &nbsp;
+    // which will show as hidden in html and api will follow thru
+    // with the update since there is a value
+    const newReviewParams = { body: reviewBody || '&nbsp;' };
```

Comment:
I don't think we want to do this. We actually want to store an empty string, rather than `&nbsp;`. I think this is going to have to be dealt with in the saga or api file, rather than in this component. Can you take a look at doing that and see if you can make it work with an empty string?

Reply:
@bobsilverberg - thanks for the feedback. I will look into more 🔍